.storybook/locale.ts

```diff
@@ -0,0 +1,223 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
we should at some point standardize how we deal with locale both brave-core and brave-ui

Reply:
Yes that would be great. Just defining the strings once would be good. I thought perhaps webpack could take all the GRD strings that are already defined and output them in to a JSON file for storybook. However, it gets complicated when I realized that there are many different GRD files. Any suggestions?

Reply:
I like the idea. however, as you said, locale can be complicated as it also goes to third-party software for translators and I'm not sure how it works. but I guess what I'm looking for is a way to simplify adding new strings so non-dev folks could change the content. or at least finding a way to avoid writing for brave-ui and brave-core. we can elaborate it later in a future chat or PR